main.go

```diff
@@ -91,8 +93,12 @@ func handleRoot(w http.ResponseWriter, _ *http.Request) {
 func main() {
 	flag.Parse()
 
-	if *disableGKECollector && *enableExtendedMetricsGKECollector {
-		log.Println("[main] `--enabledExtendedMetricsGKECollector` has no effect because `--disableGKECollector=true`")
+	if *disableGKECollector {
+		if *enableExtendedMetricsGKECollector {
+			log.Println("[main] `--collector.gke.extendedMetrics.enable` has no effect because `--collector.gke.disable=true`")
+		} else if (*gkeExtraLabelsClusterInfo != "" || *gkeExtraLabelsNodePoolsInfo != "") {
+			log.Println("[main] `--collector.gke.extendedMetrics.extraLabelsClusterInfo` and `--collector.gke.extendedMetrics.extraLabelsNodePoolsInfo` has no effect because `--collector.gke.extendedMetrics.enable=true`")
```

Comment:
This should be just an `if` ... `if *disableGKECollector` then `if (*gkeExtraLabelsClusterInfo!="" ...)` and the message is `" .... has no effect because --collector.gke.disable=true"`

Reply:
gkeExtraLabelsClusterInfo` and `gkeExtraLabelsNodePoolsInfo` should only take effect if `enableExtendedMetricsGKECollector` is enabled.

So, I don't understand how you intend to make this check simpler.

Reply:
I think it should be [this](https://github.com/DazWilkin/gcp-exporter/commit/3f8284d805cabaa2bb3deca6c778b62e12eadbc5)